charts/bindplane/templates/bindplane.yaml

```diff
@@ -368,19 +368,10 @@ spec:
         {{- if eq (include "bindplane.deployment_type" .) "StatefulSet" }}
         {{- if and (.Values.prometheus.enable) (.Values.prometheus.enableSideCar) }}
         - name: prometheus
-          image: prom/prometheus:v2.47.2
+          image: ghcr.io/observiq/bindplane-prometheus:{{ include "bindplane.tag" . }}
           ports:
             - containerPort: 9090
               name: http
-          args:
```

Comment:
Args do not need to be set here anymore, they are baked into the bindplane-prometheus image. see the bottom of this pr https://github.com/observIQ/bindplane-op-enterprise/pull/2084/files